device/samples/typescript/device_through_proxy.ts.txt

```diff
@@ -0,0 +1,108 @@
+// Copyright (c) Microsoft. All rights reserved.
```

Comment:
i don't think this is supposed to be .ts.txt filetype

Reply:
yeah I could not get that file to run, i will remove it for now